testing/architecture-test/src/changes/archunit-store/provider-task-properties.txt

```diff
@@ -141,6 +141,8 @@ Method <org.gradle.api.tasks.diagnostics.AbstractDependencyReportTask.getRendere
 Method <org.gradle.api.tasks.diagnostics.AbstractDependencyReportTask.getTaskConfigurations()> does not have raw return type (org.gradle.api.artifacts.ConfigurationContainer) assignable to any of [Provider] in (AbstractDependencyReportTask.java:0)
 Method <org.gradle.api.tasks.diagnostics.AbstractReportTask.getOutputFile()> does not have raw return type (java.io.File) assignable to any of [Property, MapProperty, ListProperty, SetProperty] in (AbstractReportTask.java:0)
 Method <org.gradle.api.tasks.diagnostics.AbstractReportTask.getProjects()> does not have raw return type (java.util.Set) assignable to any of [Property, MapProperty, ListProperty, SetProperty] in (AbstractReportTask.java:0)
+Method <org.gradle.api.tasks.diagnostics.ArtifactTransformsReportTask.getReports()> does not have raw return type (org.gradle.api.reporting.ReportContainer) assignable to any of [Provider] in (ArtifactTransformsReportTask.java:0)
```

Comment:
💭 Mirroring existing reporting task structure.  We should change these all at once.